packages/react-magma-dom/src/components/Table/TablePagination.test.js

```diff
@@ -77,11 +77,16 @@ describe('Table Pagination', () => {
       );
       const rowsSelect = getByTestId('rowPerPageSelect');
 
-      fireEvent.change(rowsSelect, { target: { value: 20 }});
+      const appliedSelection = document.querySelector(
+        'select[data-testid=rowPerPageSelect]'
+      );
+
+      fireEvent.change(rowsSelect, { target: { value: 20 } });
 
       expect(handlePageChange).toHaveBeenCalledWith(expect.any(Object), 1);
       expect(handleRowsPerPageChange).toHaveBeenCalledWith('20');
       expect(getByText(/1-20/i)).toBeInTheDocument();
+      expect(appliedSelection).toHaveDisplayValue('20');
```

Comment:
Is there a reason we are using `appliedSelection` here instead of `rowsSelect`? both are searching for the item with test id `rowPerPageSelect`

Reply:
The `document.querySelector` method is able to see the change, where `getByTestId` does not.